### PR TITLE
Add cyeva

### PR DESCRIPTION
**Pull request by Clarmy**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Clarmy** [comment]
@hhuangwx Hi! Can you take a moment to review this recipe for me? Thank you very much.

**Comment by Clarmy** [comment]
@conda-forge/help-python please review

File: recipes/cyeva/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{% set name = "cyeva" %}
+{% set version = "0.1.0.beta.3" %}
```
[the remark this answers is not in the document]
Comment (Clarmy) [reply recipes/cyeva/meta.yaml:2]: Not yet.

**Review by carterbox — CHANGES_REQUESTED** [review]
The [logs show you are installing an external tests module](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=534047&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1155) with this package. Do not install tests outside of your package.

You probably need to exit this find_packges to only include `cyeva`.
https://github.com/caiyunapp/cyeva/blob/c6cba335d2eec7bc62629fdf78ed830606046607/setup.py#L41

**Comment by Clarmy** [comment]
@carterbox I modified the setup.py to exclude `tests`. The [latest log](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=541328&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1059) shows it isn't installed.

**Comment by Clarmy** [comment]
@conda-forge/help-python  please review

**Pull request merged**